docs/source/integrating/asset-management.md

```diff
@@ -3,28 +3,85 @@ title: Asset Management
 permalink: /integrating/asset-management/
 ---
 
+## Embedding Comments on Your Site
+Talk provides an embed script that you can drop into your site where you want a comments section to appear. By default that script dynamically generate Assets
+in Talk in order to make it easier for lighter installations.
+
+You can find the embed script inside talk under `Configure > Tech Settings > Embed Script`. It should look something like this, but with your domain in place of `<TALK_ROOT_URL>`:
+```
+<div id="coral_talk_stream"></div>
+<script src="<TALK_ROOT_URL>/static/embed.js" async onload="
+  Coral.Talk.render(document.getElementById('coral_talk_stream'), {
+    talk: '<TALK_ROOT_URL>'
+  });
+"></script>
+```
+
+## Triggering the Comments Section (client side, i.e. Your site)
+
+When the embed script is triggered on your page load several things are initiated inside Talk, including fetching all comments for the specified article, establishing websocket connections for this user, and checking user’s session for SSO/authentication. 
+
+Instead of greedily triggering the embed to render on _EVERY PAGE LOAD_, we highly recommend implementing a _“lazy”_ rendering strategy to only render the comments section if a user wants to interact with it. This will greatly improve your initial page load performance, and will be critical to managing server resources if you’re running Talk on a heavy-traffic production site. 
+
+We recommend using one of these _“lazy”_ loading strategies:
+
+#### Scroll to Comments Section
+Wait for user to scroll to the comment section before triggering the embed to render. 
```

Comment:
Add a note that this requires at least v4.6.8

Reply:
will add a note here